src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralService.kt

```diff
@@ -32,17 +34,24 @@ class ReferralService(
     // fixme: hardcoded for now
     referral.referenceNumber = "HDJ2123F"
 
-    val sentReferral = repository.save(referral)
+    val sentReferral = referralRepository.save(referral)
     eventPublisher.referralSentEvent(sentReferral)
     return sentReferral
   }
 
   fun createDraftReferral(user: AuthUser, crn: String): Referral {
-    return repository.save(Referral(serviceUserCRN = crn, createdBy = authUserRepository.save(user)))
+    val interventionId = UUID.fromString("98a42c61-c30f-4beb-8062-04033c376e2d")
+    return referralRepository.save(
+      Referral(
+        serviceUserCRN = crn,
+        createdBy = authUserRepository.save(user),
+        intervention = interventionRepository.getOne(interventionId)
```

Comment:
Do we need to ensure that the intervention actually exists and throw an exception/Validation error if not?

Reply:
yes probably on the first part. but we can do that in the controller (i.e. validate the request body)

on the second part (contractual validity). i'm not sure if we need to do that. at least not right now.

@sldblog any thoughts?

Reply:
adding code to parse this in the controller so i can do the first part

Reply:
> does the intervention contract need to be valid - i.e. check contract dates

👍 probably in a separate PR though

I guess we could also remove the start/end fields... I forgot that if we don't have testable behaviour against it, we shouldn't have data